src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -56,7 +56,7 @@ public static int CopyTo(this IMemoryList<byte> list, Span<byte> destination)
             {
                 var current = list.Memory.Span;
                 var index = current.IndexOf(value);
-                if (index != -1) return Collections.Sequences.Position.Create(list, index);
+                if (index != -1) return new Collections.Sequences.Position(list, index);
```

Comment:
`System.Collections.Sequences` is imported, is there a name clash? 

i.e. can this be just `Position`?

Reply:
Ah, forgot about it. I will remove.